docs/connectors/adding-connectors-addon/README.adoc

```diff
@@ -0,0 +1,189 @@
+:quarkus-url-registry: registry/quarkus-registry/README.adoc
+:rhoas-cli-getting-started-url-registry: registry/rhoas-cli-getting-started-registry/README.adoc
+:access-mgmt-url-registry: registry/access-mgmt-registry/README.adoc
+:content-rules-registry: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817[Supported Service Registry content and rules]
+:service-binding-url-registry: registry/service-binding-registry/README.adoc
+
+//OpenShift Connectors
+:product-long-connectors: OpenShift Connectors
+:service-url-connectors: https://console.redhat.com/application-services/connectors
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-adding-connectors-addon"]
+= Adding the Red Hat {product-long-connectors} add-on to an OpenShift Dedicated trial cluster 
+ifdef::context[:parent-context: {context}]
+:context: adding-connectors-addon
+
+[role="_abstract"]
+This document describes how to add Red Hat OpenShift Connectors to an OpenShift Dedicated (OSD) Customer Cloud Subscription (CCS) trial cluster in your own OSD environment.
+
+The OSD CCS trial has a 60-day lifespan and the number of Connectors instances that you create is not limited. The trial environment is useful when you want to create Proof of Concept solutions.
+
+NOTE: Alternatively, for quick access, you can try OpenShift Connectors on the Service Preview Evaluation site.  The evaluation site involves 48 hour expiration windows and limits you to four Connectors instances at a time, as described in Service Preview guidelines.
+
+.Prerequisites
+* You must install the OpenShift Connectors service on an OSD trial cluster. 
+* You must have an existing Amazon Web Service (AWS) account.
+* Your trial cluster must use CCS on your existing AWS account. For information, see About the OpenShift Dedicated Trial. For more information on CCS, see Customer Cloud Subscriptions on AWS .
+* Your trial cluster cannot have the strimzi operator nor the Camel K operator installed (to avoid possible conflicts with the strimzi and Camel K operators that are installed as part of the Connectors Add-on).
+* You have cluster administrator permissions for your OSD account    
+* You downloaded and installed the following command line (CLI) tools:
++
+`curl`
++
+`jq`
+
+[id="proc-request-connectors-namespace"]
+== Request a Connectors namespace for your trial OSD cluster
+
+A Connectors namespace hosts your Connector instances.
+
+.Procedure
+
+. Login to the Hybrid Cloud Console (https://console.redhat.com/openshift) and then navigate to the Downloads page.
+. Scroll down to the bottom of the page and then click View API Token.
+. Click Load Token and then follow the 'Using your token in the command line' instructions to install the ocm command-line tool and run the authentication command in a terminal window.
+. In your terminal window, set the following environment variable to the URL for the Red Hat OpenShift Connecters API:
++
+----
```

Comment:
Suggest formatting these code blocks as source blocks. That's what the other app services have done. For example:

[source]
----

----